views/about.twig

```diff
@@ -5,171 +5,182 @@
+            <li><a href="https://twitter.com/rossey">Alex Ross / @rossey</a></li>
+            <li><a href="https://twitter.com/hughbertd">Hugh Downer / @HughbertD</a></li>
+            <li><a href="https://twitter.com/fully_baked">Dave Baker / @fully_baked</a></li>
+        </ul>
+    </div>
+    <div class="wrapper clearfix document">
+        <h3><a name="meetings">Meetings</a></h3>
 
-            <p>We meet once a month at <a href="https://www.google.co.uk/maps/place/St+Ambrose+Church/@50.7176004,-1.9018998,17.11z/data=!4m2!3m1!1s0x4873a1a57458cc11:0xf5803aef30b9ec8b">St. Ambrose Church hall in Westbourne</a>,
-            usually on the first Monday (although we shift to the second Monday for bank holidays) from 7.30pm.
-            There are typically two talks and then we go to the pub. There’s always tea and coffee at St. Ambrose, and usually there’ll be pizza; beer is limited to the pub afterwards.</p>
+        <p>We meet once a month at <a href="https://www.google.co.uk/maps/place/St+Ambrose+Church/@50.7176004,-1.9018998,17.11z/data=!4m2!3m1!1s0x4873a1a57458cc11:0xf5803aef30b9ec8b">St. Ambrose Church hall in Westbourne</a>,
+        usually on the first Monday (although we shift to the second Monday for bank holidays) from 7.30pm.
+        There are typically two talks and then we go to the pub. There’s always tea and coffee at St. Ambrose, and usually there’ll be pizza; beer is limited to the pub afterwards.</p>
 
-            <p>To stay informed about our upcoming meetings you can:
-                <ul>
-                    <li>follow <a href="https://twitter.com/phpdorset">@phpdorset</a></li>
-                    <li>follw us on <a href="http://www.eventbrite.co.uk/o/phpdorset-5284449005">Eventbrite</a></li>
-                    <li>check out <a href="https://webchat.freenode.net/">IRC</a> #phpdorset</li>
-                </ul>
-            </p>
+        <p>To stay informed about our upcoming meetings you can:</p>
+        <ul>
+            <li>follow <a href="https://twitter.com/PHP Dorset">@PHP Dorset</a></li>
+            <li>follw us on <a href="http://www.eventbrite.co.uk/o/PHP Dorset-5284449005">Eventbrite</a></li>
+            <li>check out <a href="https://webchat.freenode.net/">IRC</a> #PHP Dorset</li>
+        </ul>
+
+    </div>
+    <div class="wrapper clearfix document">
 
-            <h3><a name="talks">Talks</a></h3>
+        <h3><a name="talks">Talks</a></h3>
 
-            <p>Most of our meetings have talks from community members and we’re always looking for people to give a talk.
-            If you’d like to give a talk you should get in touch with us (@phpdorset, phpdorset@gmail.com) and suggest it.</p>
+        <p>Most of our meetings have talks from community members and we’re always looking for people to give a talk.
+        If you’d like to give a talk you should get in touch with us (@PHP Dorset, PHP Dorset@gmail.com) and suggest it.</p>
```

Comment:
At a guess, your find and replace to put a space in PHPDorset has affected the Twitter handle and Email address